actionview/lib/action_view/helpers/date_helper.rb

```diff
@@ -672,8 +672,6 @@ def select_year(date, options = {}, html_options = {})
       #     <time datetime="2010-11-04T17:55:45+01:00">November 04, 2010 17:55</time>
       #   time_tag Date.yesterday, 'Yesterday'  # =>
       #     <time datetime="2010-11-03">Yesterday</time>
-      #   time_tag Date.today, pubdate: true  # =>
```

Comment:
Can we replace with some other example?

Reply:
Good point. I replaced my (apparently years out of date!) use of `pubdate="pubdate"` with `item-prop=“pubdate”` but it feels a bit much to introduce [structured data](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/itemprop) into the docs here?